src/vtex-ng-filter.coffee

```diff
@@ -157,11 +159,28 @@ angular.module('vtexNgFilter', [])
         else
           []
       else if @type is 'multiple'
+        allSelected = true
+        for item in @items
+          if !item.selected
+            allSelected = false
+        
         item for item in @items when item.selected
       else if @type is 'single'
         if @selectedItem then [@selectedItem] else []
 
+    areAllSelected: =>
```

Comment:
Probably because not my native language, but I think `allSelected` would suit better

```suggestion
    allSelected: =>
```

Reply:
That's interesting. I'll just leave it that way because I'd need to change it in the OMS UI as well. But I think that makes sense!